pom.xml

```diff
@@ -1947,7 +1947,6 @@
             <dependency>
```

Comment:
It's a `dependencyManagement` entry, since we have `netty-bom` here, I think this entry is redundant, WDYT?

Reply:
True! Nice catch I'll remove it.

Reply:
True, because version of BOM is higher. If this version was higher, then it would make some sense :)